### ISSUE TITLE
[Neo4j Desktop] `Plugins and Extensions` configuration is pointing to invalid directory

### ISSUE THREAD

**Issue by FylmTM** [neo4j/neo4j#7874]
## Guidelines
- Neo4j version: 3.0.4
- Operating system: OS X El Capitan (10.11.5)
### TL;DR;

On OS X plugins are loaded from `/Applications/Neo4j\ Community\ Edition.app/Contents/Resources/app/plugins/` directory only, because Neo4j Desktop installer is hardcoded to `plugin` folder inside distribution.

Detailed information about this issue: https://github.com/graphaware/neo4j-framework/issues/41#issuecomment-244783975
### Steps to reproduce
1. Downloaded Neo4j `.dmg` installer
2. Install Neo4j
3. Start Neo4j application
4. Open `Options...`
5. Open `Plugins and extensions` folder
6. Put any plugin here that should work (e.g. APOC procedures)
7. Restart application & start Neo4j
### Expected behavior

Plugin will be loaded and work.
### Actual behavior

Plugin is not loaded.

**Comment by mihairaulea** [comment]
I can confirm this. The same for 2.3.7.

**Comment by spacecowboy** [comment]
yes, seems sub-optimal. Thanks for the report!

**Comment by spacecowboy** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;sxpvnxez&#x2F;30-gh-7874-plugins-and-extensions-configuration-is-pointing-to-invalid-directory"><img src="https:&#x2F;&#x2F;github.trello.services&#x2F;images&#x2F;trello-icon.png" width="12" height="12"> GH# 7874 &#x60;Plugins and Extensions&#x60; configuration is pointing to invalid directory</a>

**Comment by chrisvest** [comment]
This is handled better in the new Neo4j Desktop app.
